smf/clickhouse-admin-keeper/manifest.xml

```diff
@@ -17,7 +17,7 @@
   </dependency>
 
   <exec_method type='method' name='start'
-    exec='ctrun -l child -o noorphan,regent /opt/oxide/clickhouse-admin/bin/clickhouse-admin run -c /var/svc/manifest/site/clickhouse-admin/config.toml -a %{config/http_address} -l %{config/ch_address} -b %{config/ch_binary}  &amp;'
+    exec='ctrun -l child -o noorphan,regent /opt/oxide/clickhouse-admin-keeper/bin/clickhouse-admin-keeper run -c /var/svc/manifest/site/clickhouse-admin-keeper/config.toml -a %{config/http_address} -l %{config/ch_address} -b %{config/ch_binary}  &amp;'
```

Comment:
Could I ask you to use the long version of command arguments here? ^_^

Reply:
Done in [0a2e28b](https://github.com/oxidecomputer/omicron/pull/6837/commits/0a2e28b22169bf55587164395a620fb00bef3d4c)